sklearn/ensemble/_forest.py

```diff
@@ -398,6 +398,27 @@ def fit(self, X, y, sample_weight=None):
                     FutureWarning,
                 )
 
+            if self.max_features == "auto":
+                warn(
+                    "The prior default of 'auto' for max_features is "
```

Comment:
I find the message ambiguous. I don't really think that we should speak about "default" because the only way that a user gets this warning is that it did not use the default value but explicitly provide a value (that is the same as the default one). So it could be confusing. I would find better a message along these lines:

```
`max_features="auto"` has been deprecated in 1.0 and will be removed in
1.2. To keep the past behaviour, explicitly set `max_features=1.0` or
remove this parameter as it is also the default value for
RandomForestRegressor and ExtraTreesRegressor.
```